ChatBot/utils/config.py

```
@@ -0,0 +1,61 @@
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+from dataclasses import dataclass
+from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)